docker/php/env.example

```diff
@@ -4,7 +4,7 @@ DATABASE_PORT=~
 DATABASE_NAME=symfony
 DATABASE_USER=root
 DATABASE_PASSWORD=~
-DATABASE_PATH="%kernel.project_dir%/data/db/wallabag.sqlite"
+DATABASE_PATH='"%kernel.project_dir%/data/db/wallabag.sqlite"'
```

Comment:
Before this change the generated value in parameters.yml was `database_path: %kernel.project_dir%/data/db/wallabag.sqlite` without quotes, which is invalid